packages/ketcher-react/src/script/ui/views/components/ContextMenu/ContextMenu.tsx

```diff
@@ -0,0 +1,182 @@
+/****************************************************************************
+ * Copyright 2022 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************/
+
+import { Action, fromBondsAttrs, fromOneBondDeletion } from 'ketcher-core'
+import { useCallback } from 'react'
+import type { ItemParams, PredicateParams } from 'react-contexify'
+import { Item, Menu, Submenu } from 'react-contexify'
+import 'react-contexify/ReactContexify.css'
+import { useAppContext } from 'src/hooks'
+import Editor from 'src/script/editor'
+import tools from 'src/script/ui/action/tools'
+import Icon from 'src/script/ui/component/view/icon'
+import styles from './ContextMenu.module.less'
+
+export const CONTEXT_MENU_ID = 'KeTcHeR-CoNtExT-MeNu'
```

Comment:
Just interesting, why are we using id with such case? :) 
can't it be `ketcher-context-menu`
or
`ketcherContextMenu`?

Reply:
It can be `ketcher-context-menu` or `ketcherContextMenu`.
Just because I wanted it unique, in case there would be other context menus. But now I'm thinking if there are more context menus we'd better use enums to manage the IDs.

Reply:
Let's use enum 😄 
And in this case, I guess, we can name like `ketcherBondContextMenu`. So we use `ketcher` in order to avoid conflicts with other libraries/tools and `bond` to specify its purpose.